lib/LLVMIRCodeGen/GlowJIT.cpp

```diff
@@ -150,24 +131,12 @@ GlowJIT::GlowJIT(llvm::TargetMachine &TM)
             return nullptr;
           },
           [](Error Err) { cantFail(std::move(Err), "lookupFlags failed"); })),
-#if LLVM_VERSION_MAJOR < 8 || FACEBOOK_INTERNAL
       objectLayer_(ES_,
                    [this](llvm::orc::VModuleKey) {
                      return RTDyldObjectLinkingLayer::Resources{
                          std::make_shared<SectionMemoryManager>(), resolver_};
                    },
                    NotifyLoadedFunctor(this)),
-#else
```

Comment:
Why did you remove this snippet? This was introduced for LLVM >=8, IIRC.

Reply:
This passed in the internal version, therefore I wrongfully assumed it passes for llvm8. It doesn't built locally. reverting all of these patches.